src/create/create.nim

```diff
@@ -13,6 +13,10 @@ proc create*(conf: Conf) =
         let msg = &"Both --approach and --article were provided. Please specify only one."
         stderr.writeLine msg
         quit QuitFailure
+      if conf.action.difficulty.isSome:
+        let msg = "The difficulty argument is not supported for approaches"
+        stderr.writeLine msg
+        quit QuitFailure
```

Comment:
Does this need to be a failure, or is a warning sufficient?

Reply:
We tend to error for arguments that are not supported.